srcpkgs/swapspace/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'swapspace'
+pkgname=swapspace
+version=1.17
+revision=1
+wrksrc=Swapspace-${version}
+build_style=gnu-configure
+hostmakedepends="autoconf automake"
+short_desc="Dynamic swap manager for Linux"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/Tookmund/swapspace"
+changelog="https://raw.githubusercontent.com/Tookmund/Swapspace/master/NEWS"
+distfiles="https://github.com/Tookmund/Swapspace/archive/v${version}/${version}.tar.gz"
+checksum=5a2d82ae7922588bf0c3e3728248672b94666bab76b4be569d3a8793edfabb4f
+
+pre_configure() {
+	autoreconf -vfi
+}
+
+post_install() {
+	vman doc/swapspace.8
+
```

Comment:
what is the newline here for?

Reply:
My personal preference is to split the file installations and the service installations visually